app/enterprise/0.34-x/kong-aws.md

```diff
@@ -0,0 +1,195 @@
+
+### ASG
+
+For fault tolerance, we recommend that auto-scaling groups are configured to launch in multiple availability zones. By registering ASG instances with an ELB, Kong can be scaled horizontally simply by adjusting the size of the scaling group.
+
+### Security Groups
+
+#### Kong Security Group
+
+Following the general steps to create a Security Group, as outlined in the AWS Documentation, perform these steps to create the necessary security groups for Kong instances:
+
+1. Open the Amazon VPC console at https://console.aws.amazon.com/vpc/.
+
+2. In the navigation pane, choose Security Groups.
+
+3. Choose Create Security Group.
+
+4. Enter a name of the security group, such as 'Kong Nodes' and provide a description of your choice.
+
+5. Select the ID of your VPC from the VPC menu.
+
+6. Use the Add Rule button to add the following Inbound rules:
+
+Type - Protocol - Port Range - Source - Description
+Custom TCP Rule - TCP - 8000-8003 - <VPC CIDR> - Kong proxy, Admin API, Manager, and Dev Portal (HTTP)
+Custom TCP Rule - TCP - 8443-8446 - <VPC CIDR> - Kong proxy, Admin API, Manager, and Dev Portal (HTTPS)
+
+#### RDS Security Group
+
+Following the general steps to create a Security Group, as outlined in the AWS Documentation, perform these steps to create the necessary security groups for the RDS instance:
+
+1. Open the Amazon VPC console at https://console.aws.amazon.com/vpc/.
+
+2. In the navigation pane, choose Security Groups.
+
+3. Choose Create Security Group.
+
+4. Enter a name of the security group, such as 'Kong Nodes' and provide a description of your choice.
+
+5. Select the ID of your VPC from the VPC menu.
+
+6. Use the Add Rule button to add the following Inbound rules:
+
+Type - Protocol - Port Range - Source - Description
+Custom TCP - TCP - 5432 - <VPC CIDR> - Postgres access for Kong nodes
+
+### RDS
+
+For fault tolerance, we recommend that PostgreSQL RDS instances are created with multi-AZ availability. Disk encryption for RDS instances is encouraged for sensitive environments, but is not a requirement for running Kong.
+
+## Kong Configuration
+
+## Installation
+
+Refer to the existing generic documentation on [installing Kong](https://konghq.com/install/).
+
+Additional directions on installing Kong Enterprise on AWS Linux can be found [here](https://docs.konghq.com/enterprise/0.34-x/installation/amazon-linux/).
+
+### Proxy protocol
+
+When running Kong behind a load balancer forwarding requests via the Proxy protocol, we recommend configuring Kong to listen for Proxy protol requests in order to correctly identify the downstream client address. This can be set by defining the `proxy_protocol` stanza of the `proxy_listen` and `admin_listen` directives as outlined in the [configuration reference](https://docs.konghq.com/1.0.x/configuration/#proxy_listen):
```

Comment:
```suggestion
When running Kong behind a load balancer forwarding requests via the Proxy protocol, we recommend configuring Kong to listen for Proxy protocol requests in order to correctly identify the downstream client address. This can be set by defining the `proxy_protocol` stanza of the `proxy_listen` and `admin_listen` directives as outlined in the [configuration reference](https://docs.konghq.com/1.0.x/configuration/#proxy_listen):
```